srcts/src/shiny/shinyapp.ts

```diff
@@ -614,7 +614,7 @@ class ShinyApp {
 
       const nsPrefix = el.attr("data-ns-prefix") as string;
       const nsScope = this._narrowScope(scope, nsPrefix);
-      const show = condFunc(nsScope);
+      const show = !!condFunc(nsScope);
```

Comment:
I'd rather use the class instead of the accepted workaround of `!!`.

```suggestion
      const show = Boolean(condFunc(nsScope));
```

Reply:
Agreed, thanks!